thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/CompositePipelineConfigTranslator.java

```diff
@@ -179,7 +179,8 @@
   private static final Set<String> TUNING_OFF_COMPONENTS =
       ImmutableSet.of("MIGRATED_ALGORITHM_FILTER", "MIGRATED_ALGORITHM", "MIGRATED_ALGORITHM_BASELINE");
   private static final Map<String, String> DETECTOR_TO_BASELINE =
-      ImmutableMap.of("ALGORITHM", "ALGORITHM_BASELINE", "MIGRATED_ALGORITHM", "MIGRATED_ALGORITHM_BASELINE");
+      ImmutableMap.of("ALGORITHM", "ALGORITHM_BASELINE", "MIGRATED_ALGORITHM", "MIGRATED_ALGORITHM_BASELINE",
```

Comment:
I'm thinking to generalize the multiple inheritance approach (baseline provider and detector implemented by the same class) in all detector implementations, as you did in this Hot Winters implementation. After that, hard code this shouldn't be necessary anymore. So this hack should go away soon ^^. Thank you for making a great example!

Reply:
Yes this is hacky and let's think of how to get rid of it.